src/MVVM/PropertyChanged/BindingBase/BindingBase.cs

```diff
@@ -58,10 +65,45 @@ private void GetLogServiceInstance()
                 .GetValue(
                     this);
 
-            if(logService == null)
-                throw new InvalidEnumArgumentException();
 
-            _logService = logService;
+            _log = logService ?? throw new InvalidEnumArgumentException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5640474)